src/main/views/MattermostBrowserView.ts

```diff
@@ -89,6 +91,12 @@ export class MattermostBrowserView extends EventEmitter {
         if (process.platform !== 'darwin') {
             this.browserView.webContents.on('before-input-event', this.handleInputEvents);
         }
+        this.browserView.webContents.on('input-event', (_, inputEvent) => {
```

Comment:
This was transferred over from the preload script, that checks when the user clicks on the main view to close any of the dropdowns.